lib/ParamedicConfig.js

```diff
@@ -36,8 +36,10 @@ ParamedicConfig.parseFromArguments = function (argv) {
         startPort:         argv.startport || argv.port,
         endPort:           argv.endport || argv.port,
         externalServerUrl: argv.externalServerUrl,
-        reportSavePath:    !!argv.reportSavePath? argv.reportSavePath: undefined,
-        cleanUpAfterRun:   !!argv.cleanUpAfterRun? true: false
+        outputDir:         !!argv.outputDir? argv.outputDir: null,
+        cleanUpAfterRun:   !!argv.cleanUpAfterRun? true: false,
+        logMins:           !!argv.logMins? argv.logMins: null,
```

Comment:
It might be a good idea to initialize defaults for optional parameters here e.g. util.DEFAULT_LOG_TIME